data/model/user.py

```diff
@@ -111,6 +111,34 @@ def create_user(
     return created
 
 
+def create_init_user(
```

Comment:
Can we add a test for this? Just to test the new constraints we're adding to `create_user`.